src/maud/io.py

```diff
@@ -45,26 +46,6 @@
 from maud.utils import codify
 
 
-MECHANISM_TO_PARAM_IDS = {
```

Comment:
Don't know how this survived so long!

Reply:
Hope that we would bring them back I suppose